mne/datasets/bnci/bnci.py

```diff
@@ -0,0 +1,608 @@
+# Author: Alexandre Barachant <alexandre.barachant@gmail.com>
+# License: BSD Style.
+
+import os
+from os import path as op
+import numpy as np
+
+from scipy.io import loadmat
```

Comment:
I see. I have to import within the function ...
Out of curiosity, what's the reason for using nested import for scipy.io ?